.github/workflows/build-custom-docker-image.yml

```diff
@@ -0,0 +1,38 @@
+name: OpenSearch Dashboards Build Custom Docker Image
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Build-Custom-Docker-Image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: append config and custom command
+        env:
+          TRACKING_ID: ${{secrets.GA_TRACKING_ID}}
+          PLUGINS: ${{secrets.PLUGINS}}
+        run: |
+          if [[ $TRACKING_ID ]]; then
+            echo -e "\r\n" >> config/playground/docker/dev/extra_config/osd.yml
+            echo google_analytics_plugin.trackingID: $TRACKING_ID >> config/playground/docker/dev/extra_config/osd.yml
```

Comment:
Non blocking but just curious, are we expecting more custom plugins to be implemented following the similar practice as google analytics ? Maybe it's worth to create another env variable grouping all potential custom plugins together then just do an iteration like 

`for plugin in $(Plugins -e); do`

Reply:
good suggestion, @BionIT could we create a issue to follow up